middleware/confidenceScore.js

```diff
@@ -11,24 +11,24 @@
  * - detection (or specification) of query type. i.e. an address shouldn't match an admin address.
  */
 
+const _ = require('lodash');
 const stats = require('stats-lite');
 const logger = require('pelias-logger').get('api');
-const check = require('check-types');
 const field = require('../helper/fieldValue');
 
 var RELATIVE_SCORES = true;
 
 function setup(peliasConfig) {
-  if (check.assigned(peliasConfig)) {
+  if (!_.isNil(peliasConfig)) {
     RELATIVE_SCORES = peliasConfig.hasOwnProperty('relativeScores') ? peliasConfig.relativeScores : true;
```

Comment:
can't believe you didn't rewrite this line :P

lots of ways to make it better and shorter with `peliasConfig.get`

Reply:
I was just trying to do a no-op, was tempted in a few places to change a few things but resisted!